#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_ethernet-manpagefix.dpatch by William Vera <billy@billy.com.mx>
##
## DP: Character cleanup for UTF-8 compatibility.

@DPATCH@
diff -urNad nemesis-1.4~/man/nemesis-ethernet.1 nemesis-1.4/man/nemesis-ethernet.1
--- nemesis-1.4~/man/nemesis-ethernet.1	2008-07-02 05:34:33.000000000 -0500
+++ nemesis-1.4/man/nemesis-ethernet.1	2008-07-02 05:34:53.000000000 -0500
@@ -47,7 +47,7 @@
 .I payload-file
 as the payload when injecting Ethernet frames.  The maximum payload size is 
 1500 bytes in order to account for the maximum Ethernet frame size.  Payloads
-can also be read from stdin by specifying '-P -'
+can also be read from stdin by specifying '\-P \-'
 instead of a payload-file.
 
 The payload file can consist of any arbitrary data though it will be most useful
